### ISSUE TITLE
Preset Theme link not working

### ISSUE THREAD

**Issue by Purus** [pbernasconi/ionic-theme-editor#8]
When clicking on the "Preset Theme" button, nothing happens.

**Comment by madfriend** [comment]
Bump

**Comment by kevbost** [comment]
@madfriend I agree.  This is a great tool and would be very useful for me right now, alas it is partially  broken if navigation does not work.

**Comment by TimoSolo** [comment]
Looks like there aren't any preset themes :(
looking at the code, the ng-click is commented out and there is no page for presets. Would be nice..

<a ng-class="{'selected' : isButtonBar(1)}">Preset themes</a>
